common/src/main/java/io/netty/util/concurrent/EventExecutorGroup.java

```diff
@@ -27,7 +29,7 @@
  * life-cycle and allows shutting them down in a global fashion.
  *
  */
-public interface EventExecutorGroup extends ScheduledExecutorService, Iterable<EventExecutor> {
```

Comment:
This is a breaking change, we can't add new methods to `EventExecutorGroup`

Reply:
Incredibly it hasn't detected as breaking change by the checks...

Reply:
@franz1981 I think you managed to outsmart the checks by adding a new superinterface :)

Reply:
Argh! Will fix this today